spec/prog/vnet/subnet_nexus_spec.rb

```diff
@@ -118,8 +120,9 @@
     it "hops back to wait if nics are done" do
       expect(ss).to receive(:set?).and_return(false)
       expect(SemSnap).to receive(:new).with(nic.id).and_return(ss)
-      expect(ps).to receive(:nics).and_return([nic])
+      expect(ps).to receive(:nics).and_return([nic]).twice
```

Comment:
`.twice` is fine, but `.at_least(:once)` is my go-to, since usually in such a situation I mostly want to prune mocks that get used *zero* times. This is why I ignore the standard advice about "oh use 'allow' if you are indifferent" because it tends to accrue forever, so I use it selectively (e.g. shared fixtures that condense nested contexts enough to be worth code that degrades to the extraneous and misleading)